graphstorm-processing/docker/0.4.0/emr-serverless/Dockerfile.cpu

```diff
@@ -1,5 +1,5 @@
 ARG ARCH=x86_64
-FROM public.ecr.aws/emr-serverless/spark/emr-7.1.0:20240528-${ARCH} as base
+FROM public.ecr.aws/emr-serverless/spark/emr-7.3.0:20241008-${ARCH} as base
```

Comment:
Do we want to move with the latest here: public.ecr.aws/emr-serverless/spark/emr-7.6.0:20250107?

Reply:
I have not tested that version yet, let's move that to 0.4.1?